theano/__init__.py

```diff
@@ -31,10 +31,6 @@
 
 import sys
 
-if sys.platform == 'win32' and sys.version_info[0:2] == (3, 5):
```

Comment:
Here is an error report with WinPython 3.5 on windows:

https://github.com/Theano/Theano/issues/4664#issuecomment-263134608

I don't have objection to removing the check, but I fear that there is still some python 3.5 on windows that have problems. conda and the official python are fine from what you tell. Do you have an idea why winpython have a problem?

People with python 3.5 not support will still get an error, just an error very hard to understand. If you have an idea how we could detect here if we support the current Python 3.5 on Windows, it would be great.

Otherwise, I think we will keep this and probably add a note in the installation of windows that it is not all python 3.5 that are supported.